src/ioc/Ioc.js

```diff
@@ -0,0 +1,35 @@
+let services = {};
```

Comment:
You don't need to put this here. You can move it inside the constructor as it was before

Reply:
But if I have it inside, would I be able to modify it (register services) having the `Object.freeze()` on the instance?

Reply:
It works having it inside the constructor. Changed it now.